content/02.introduction.md

```diff
@@ -15,6 +15,7 @@ There are also potential interaction effects between gender and race or ethnicit
 Another recent analysis found that minority scientists tend to apply for awards on topics with lower success rates [@doi:10.1126/sciadv.aaw7238].
 This finding might be the result of minority scientists selecting topics in more poorly funded areas.
 Alternatively, reviewing scientists may not recognize the scientific importance of these topics, which may be of particular interest to minority scientists.
+One way to address these competing hypotheses is to directly examine peer recognition.
```

Comment:
We have a similar sentence at the end of the first paragraph. Should we keep both?

Reply:
Ahh - ok! I missed it.